src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;

```

Comment:
The 'quantity' variable should be checked if it's a positive number. Negative values for quantity might not make sense in this context.